README.md

```diff
@@ -469,6 +469,7 @@ To add a resource or contribute, please check out our contribution guide [here](
 - [React Patterns](https://reactpatterns.com/)
 - [Useful Packages you should know in React](https://medium.com/better-programming/14-useful-packages-every-react-developer-should-know-55b47a325d3)
 - [Learn React Test Driven Development](https://learntdd.in/react/)
+- [React JS Clones of famous websites - YouTube](https://youtube.com/playlist?list=PLf16UKl7nR5AjcrYOe1niifJSAls3spDk)
```

Comment:
Please add the title of the playlist as it is

Reply:
Ok sir I have done that in another pull request. Please close this old pull request. Thank you for being so supportive and making this unique repo, which can help beginners like us contribute to open-source.